main.go

```diff
@@ -129,13 +145,6 @@ func main() {
 }
 
 func initFlags(fs *pflag.FlagSet) {
-	fs.StringVar(
```

Comment:
this shouldn't be removed like this, lets deprecate it and remove in the next release with an appropriete TODO

Reply:
lets add some logic to handle the case properly when user submits this flag vs the `diagnostics-address`

Reply:
Addressed them, PTAL